### DISCUSSION TITLE
The icon for removing repository topic isn't centered in the Chrome

### DISCUSSION THREAD

**Discussion by magicmatatjahu** [community/community#38672] [Repositories]
> If this category for raising UI issues is not appropriate, I apologize and please let me know where I should create it. Thanks!

In the Chrome the icon for removing repository topic isn't centered. Take a look at the screenshots.

Chrome - 107.0.5304.87:

![image](https://user-images.githubusercontent.com/20404945/200844497-3d7df287-5a43-414a-9e30-b17dc3828536.png)

Firefox - 105.0.3:

![image](https://user-images.githubusercontent.com/20404945/200844079-39d80ac4-d52a-45a6-8aba-10389d68ba46.png)

I didn't test other browsers.

**Comment by ossanna16** [comment]
Hi there @magicmatatjahu and welcome to our community! Thank you for reporting this!
